### ISSUE TITLE
Support for 120hz position update

### ISSUE THREAD

**Issue by lomeat** [dmarcotte/easy-move-resize#97]
I have a macbook m1 pro with 120hz and display with 144hz screen update, so the current position updating looks like "lagging" because of 60 "fps".

The best option is take the monitor refreshing latency and use it to delay calculations (as a delta) 
Or you can add variants to menu for choosing user frequency

**Comment by RealA10N** [comment]
Hi! I have the same issue.
It seems like modifiying the following two lines in the source code to something like 1/144 will do the trick. 
https://github.com/dmarcotte/easy-move-resize/blob/cf7c50e5c32982c3dc7542096974e24b66ec5c4e/easy-move-resize/EMRAppDelegate.h#L5

**Comment by RealA10N** [comment]
Here is the version I compiled, it works great, much smooooooother!
[Easy.Move+Resize.app.zip](https://github.com/dmarcotte/easy-move-resize/files/12754419/Easy.Move%2BResize.app.zip)
(unzip, then drap to applications, then launch).

**Comment by mularkey** [comment]
This! Incredible, thank you!

**Comment by Arecsu** [comment]
Indeed. It works so much better @RealA10N, thank you! 😊

The stutter in the main release also happens in a 60hz screen. It looks like 20fps, maybe less. I'm using a Macbook Air with Sonoma

**Comment by jjolmo** [comment]
Works great there. Could be ideal to be able to configure the refresh rate via options

**Comment by seanlzq-hm** [comment]
> @RealA10N Works great!. It could be ideal to be able to configure the refresh rate via options

Yes this would be great. I'm using a 144hz monitor as well and I'm notifcing the jankiness still

**Comment by RealA10N** [comment]
> > @RealA10N Works great!. It could be ideal to be able to configure the refresh rate via options
> 
> 
> 
> Yes this would be great. I'm using a 144hz monitor as well and I'm notifcing the jankiness still 

Hi, you can change the constant to 1/144 or even 1/200 and compile the project yourself 🙂

I'm not the maintainer of this project, it's just a quick fix

**Comment by seanlzq-hm** [comment]
> > > @RealA10N Works great!. It could be ideal to be able to configure the refresh rate via options
> > 
> > 
> > Yes this would be great. I'm using a 144hz monitor as well and I'm notifcing the jankiness still
> 
> Hi, you can change the constant to 1/144 or even 1/200 and compile the project yourself 🙂
> 
> I'm not the maintainer of this project, it's just a quick fix

I would if I knew how, I'm just the average consumer with no app development knowledge 😅

**Comment by csumpasd** [comment]
@seanlzq-hm you have to download the project (for example, through the releases page, it's a zip file), then open the contained .xcodeproj file in xcode, find the EMRAppDelegate.h file on the right and edit the variables inside, and then do the steps at the bottom of the readme file, starting with "In Xcode...". You should have the app in your homedir in a folder.

**Comment by seanlzq-hm** [comment]
[Easy Move+Resize.zip](https://github.com/dmarcotte/easy-move-resize/files/13624119/Easy.Move%2BResize.zip)

Alright, here's what I got and currently using it for my 144hz monitor. Thanks for the help @csumpasd and @RealA10N !

**Comment by bcalik** [comment]
> Edited and recompiled for my 144hz monitor...
> 
> [Easy Move+Resize.zip](https://github.com/dmarcotte/easy-move-resize/files/13624119/Easy.Move%2BResize.zip)
> 
> So far so good. Thanks for the help @csumpasd and @RealA10N !

This works awesome on my M1 Macbook Pro (120hz) with any 144hz or 60hz displays together!

**Comment by cristianrgreco** [comment]
@dmarcotte it'd be great to make this the default or make it user configurable and get a release, i'm a bit nervous about downloading and running apps attached to issue comments

**Comment by cschep** [comment]
I've opened #102 to get moving on a menu item for this. Would love any thoughts!

**Comment by dmarcotte** [comment]
Resolved in #116
